py/picca/delta_extraction/expected_fluxes/dr16_expected_flux.py

```diff
@@ -625,184 +688,72 @@ def compute_var_stats(self, forests):
         ExpectedFluxError if wavelength solution is not valid
         """
         # initialize arrays
-        eta = np.zeros(self.num_bins_variance)
-        var_lss = np.zeros(self.num_bins_variance)
-        fudge = np.zeros(self.num_bins_variance)
-        error_eta = np.zeros(self.num_bins_variance)
-        error_var_lss = np.zeros(self.num_bins_variance)
-        error_fudge = np.zeros(self.num_bins_variance)
+        if "eta" in self.fit_variance_functions:
+            eta = np.zeros(self.num_bins_variance) + 1.
+        else:
+            eta = self.get_eta(self.log_lambda_var_func_grid)
+        if "var_lss" in self.fit_variance_functions:
+            var_lss = np.zeros(self.num_bins_variance) + 0.1
```

Comment:
The reason is that the fits are done dividing the fudge by FUDGE_REF so that its values are not as small. But good point, code them in variables at the top so that this is more clear